### ISSUE TITLE
Visual Tuning Datasets Filtering Strategy

### ISSUE THREAD

**Issue by Zxr1314** [WePOINTS/WePOINTS#25]
Will the complete filtering strategy of visual instruction tuning datasets be published soon?

**Comment by YuanLiuuuuuu** [comment]
Thank you for your interests in POINTS1.5. We will discuss the filtering strategy of visual instruction tuning data in the upcoming paper of POINTS1.5. Stay tuned!

**Comment by YuanLiuuuuuu** [comment]
@Zxr1314 The preview of POINTS1.5 [technical report](https://github.com/WePOINTS/WePOINTS/blob/main/POINTS1_5_preview.pdf) is available.

**Comment by Zxr1314** [comment]
> @Zxr1314 The preview of POINTS1.5 [technical report](https://github.com/WePOINTS/WePOINTS/blob/main/POINTS1_5_preview.pdf) is available.

Thanks very much. Another question is about model_soup strategy. The document said that models trained with different datasets are used to make model soup. How are the datasets divided? What principles are the strategy based on? Looking forward to your reply

**Comment by YuanLiuuuuuu** [comment]
Sorry for the late reply. In POINTS1.5, we choose two sets of models to apply model soup on:

1. Following [POINTS1.0](https://arxiv.org/pdf/2409.04828), for the visual instruction tuning stage, we incrementally add more datasets (each dataset denoted as \(d_i\)) into the dataset pool (denoted as \(D_i\)) used to train our model. During the dataset selection process, we obtain a series of models.
2. For each dataset pool (\(D_i\)), we train the model for one epoch and two epochs, resulting in a series of models.
3. The best-performing models from the above two steps (according to the OpenCompass score) are selected to conduct model soup.
4. We use the simplest linear model soup. Although we also tried more advanced model soup strategies, such as [TIES](https://arxiv.org/abs/2306.01708), we did not observe improvements.
